string-manipulation-1/src/is-vowel.js

```diff
@@ -0,0 +1,15 @@
+/* exported isVowel */
+function isVowel(char) {
+  var testingVariable = char.toUpperCase();
+  var upperCaseVowels = ['A', 'E', 'I', 'O', 'U'];
+  var booleanOutput = true;
+  for (var i = 0; i < upperCaseVowels.length; i++) {
+    if (testingVariable === upperCaseVowels[i]) {
+      booleanOutput = true;
+      continue;
+    } else if (testingVariable !== upperCaseVowels[i]) {
+      booleanOutput = false;
+    }
+  }
+  return booleanOutput;
+}
```

Comment:
Did you test this?
<img width="222" alt="image" src="https://user-images.githubusercontent.com/32880187/188957707-dc560bfd-a46f-40c4-bf19-db5544562ff1.png">

Reply:
Sorry! I was still in progress but didnt want to waste my progress.